about.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone Project</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <script src="https://kit.fontawesome.com/afdd353a4f.js" crossorigin="anonymous"></script>
+    <script src="./js/app.js" defer></script>
+</head>
+<body>
+    <header>
+        <nav class="desktop-version-nav">
+            <img src="./img/hamb-icon.svg" class="hamb-icon" alt="menu-bar">
+        </nav>
+        <nav class="desktop-social-icons">
+            <ul class="new-nav">
+                <li><a href="#"><img src="./img/desktop/facebook.svg" alt="facebook logo"></a></li>
+                <li><a href="#"><img src="./img/desktop/twitter.svg" alt="twitter logo"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </nav>
+        <section class="logo-company">
+            <img src="./img/logo-mother-coding.svg" alt="logo-company">
+        </section>
+        <div class="mobile-menu-page" id="mobile_menu_page">
+            <div class="effect">
+                <div class="create-x">
+                    <i class="fa-solid fa-x" id="icon-x"></i>
+                </div>
+                <ul class="mobile-links">
+                    <li><a href="#about" id="about">About</a></li>
+                    <li><a href="index.html#main-program">Program</a></li>
+                    <li><a href="index.html#future-speakers">Speakers</a></li>
+                    <li><a href="index.html#future-partners">Sponsors</a></li>      
+                </ul>
+            </div>
+        </div>
+   </header>
+   <article class="white-logo-company">
+            <figure class="logo-desktop">
+                    <a href="index.html"><img src="./img/logo-mother-coding.svg" alt="logo-company-desktop"></a>
+            </figure>
+            <ul>
+                <li class="nav-item"><a href="#">About</a></li>
+                <li class="nav-item"><a href="index.html#main-program">Program</a></li>
+                <li class="nav-item"><a href="index.html#future-speakers">Speakers</a></li>
+                <li class="nav-item"><a href="index.html#future-partners">Sponsors</a></li>
```

Comment:
> The home page should have a link in the menu to the About page and vice verse.

- [ ] As stated in the project requirement, there should be a link in the menu that links to the home page, in that note, since this is the About page, you do not need to have the `About` in the menu link list, rather it should be `Home` which links back to the index page. I'll suggest you update your code to effect this. :smile:  :wave:

Reply:
This part is also corrected.
thank you